src/runtimes/node/bundlers/esbuild/bundler.ts

```diff
@@ -29,45 +29,49 @@ const RESOLVE_EXTENSIONS = ['.js', '.jsx', '.mjs', '.cjs', '.ts', '.tsx', '.mts'
 /**
  * Our own `includedFiles` syntax is slightly different from what esbuild expects as `externals`.
  *
- * Turns !node_modules/test/** into ./node_modules/test/*
+ * Turns !node_modules/test/** into test
+ * and !lang/en.* into ./lang/en.*.
+ * ESBuild can't handle multiple globs in a single pattern, so we resolve them instead.
  */
-const includedFilesToEsbuildExternals = (
-  includedFiles: string[],
-  failOnDouble: boolean,
-  logger: Logger,
-  functionName: string,
-) =>
-  includedFiles
-    // only keep negated patterns
+const includedFilesToEsbuildExternals = async (includedFiles: string[], baseDir: string) => {
+  const exclusions = includedFiles
     .filter((pattern) => pattern.startsWith('!'))
-    // warn about invalid patterns
-    .filter((pattern) => {
-      const containsMultipleStars = pattern.indexOf('*') !== pattern.lastIndexOf('*')
+    .map((pattern) => pattern.slice(1))
+    // esbuild expects relative paths
+    .map((pattern) => `./${pattern}`)
```

Comment:
Is this going to work on Windows?

Reply:
Didn't really think about that, but I guess it does, yeah - the unit tests are all green, also on windows.